Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```
@@ -0,0 +1,204 @@
+{
+    "author": "Gavin Aguiar",
+    "name": "EventHub trigger",
+    "description": "$EventHubTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with EventHubTrigger function",
+            "input" : {
+                "userCommand" : "createNewApp",
+                "assignTo" : "$(APP_FILENAME)"
+            },
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionApp",
+                "replaceTokensInText_FunctionApp",
+                "writeFile_FunctionApp",
+                "showMarkdownPreview"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the main file",
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionBody",
+                "replaceTokensInText_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "input" : {
+                "userCommand" : "createBluePrint",
+                "assignTo" : "$(BLUEPRINT_FILENAME)"
+            },
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintFile",
+                "replaceTokensInText_BlueprintFile",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "appendFiles_Blueprint_BlueprintBody",
+                "writeFile_Blueprint",
+                "readFileContent_BlueprintImport",
+                "replaceTokensInText_BlueprintImport",
+                "appendFileContent_FunctionApp",
+                "readFileContent_BlueprintRegister",
+                "replaceTokensInText_BlueprintRegister"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the Blueprint",
+            "actions": [
+                "getBluePrintFileName",
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_BlueprintBody",
+                "replaceTokensInText_BlueprintBody",
+                "appendFileContent_Blueprint"
+            ]
+        }
+    ],
+    "actions" : [
+        {
+            "name": "getFunctionName",
+            "type": "UserInput",
+            "assignTo": "$(FUNCTION_NAME_INPUT)",
+            "paramId": "trigger-functionName",
+            "defaultValue": "EventhubTrigger"
+        },
+        {
+            "name": "getEventHubName",
+            "type": "UserInput",
+            "assignTo": "$(FUNCTION_NAME_INPUT)",

```

This should be `$(EVENTHUB_NAME)`

```suggestion
            "assignTo": "$(EVENTHUB_NAME)",
```

---

Updated to EVENTHUB_NAME